pom.xml

```diff
@@ -133,16 +133,12 @@
         </pluginManagement>
         <plugins>
             <plugin>
-                <groupId>org.wso2.siddhi</groupId>
-                <artifactId>siddhi-doc-gen</artifactId>
-                <version>${siddhi.version}</version>
-                <executions>
-                    <execution>
-                        <goals>
-                            <goal>generate-md-docs</goal>
-                        </goals>
-                    </execution>
-                </executions>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-release-plugin</artifactId>
```

Comment:
Plugin is duplicated. Can you remove that?

Reply:
Remove the duplicate plugin